Clarify install instructions (#576)
amplitude/amplitude-docs: content/collections/guides_and_surveys/en/sdk.md

### Third-party analytics provider

Using the Guides and Surveys standalone SDK with another analytics provider requires extra configuration to help map properties to Amplitude.
If you don't use the Amplitude Analytics [Browser SDK 2](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2), you can still use Guides and Surveys but you need to configure the SDK to work with your third-party analytics provider. First, add the SDK to your project using the script tag, or via npm or Yarn as outlined above.
But, instead of calling `amplitude.add(window.engagement.plugin())`, you'll need to call `init` and `boot`.

> {{partial:collapse name="Usign NPM or Yarn"}}

Author: Since `init` can now be safely called multiple times I would like to remove this as a special case and just document it generally. At some point, it would be great to not do `init` automatically with the script so people can properly use the arguments of  the method.

### Third-party analytics provider

> If you don't use the Amplitude Analytics [Browser SDK 2](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2), you can still use Guides and Surveys but you need to configure the SDK to work with your third-party analytics provider. First, add the SDK to your project using the script tag, or through npm or Yarn as outlined above.

Reviewer: ```suggestion
If you don't use the Amplitude Analytics [Browser SDK 2](/docs/sdks/analytics/browser/browser-sdk-2), you can still use Guides and Surveys but you need to configure the SDK to work with your third-party analytics provider. First, add the SDK to your project using the script tag, or through npm or Yarn as outlined above.
```

But, instead of calling `amplitude.add(window.engagement.plugin())`, you need to call `init` and `boot`.

> #### Initialize SDK

Reviewer: ```suggestion
#### Initialize the SDK
```

## Show

> Display a specific guide or survey. This will ignore any targeting rules and limits except for page targeting.

Reviewer: ```suggestion
Display a specific guide or survey. This ignores any targeting rules and limits except for page targeting.
```

## Forward event

> Forwared third-party Analytics events to the Guides and Surveys SDK to trigger guides and surveys that use the `*On event tracked*` [trigger](/docs/guides-and-surveys/guides/guides/setup-and-target#triggers).

Reviewer: ```suggestion
Forward third-party Analytics events to the Guides and Surveys SDK to trigger guides and surveys that use the `*On event tracked*` [trigger](/docs/guides-and-surveys/guides/setup-and-target#triggers).
```

```js
engagement.forwardEvent(event: Event): void
```

| Parameter | Type  | Description                                                                                                                                            |
| --------- | ----- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | `event`   | Event | Required. An [event](/docs/sdks/analytics/browser/browser-sdk-2#track-an-event) object. It will trigger a guide or survey if its `event_type` matches. |

Reviewer: ```suggestion
| `event`   | Event | Required. An [event](/docs/sdks/analytics/browser/browser-sdk-2#track-an-event) object. It triggers a guide or survey if its `event_type` matches. |
```
